stanford_soe_r25.feed.inc

```diff
@@ -0,0 +1,203 @@
+<?php
+
+// takes ajax requests from FullCalendar, passes them on to the 25Live web api, and returns the results as JSON
+
+// bit of DRY code to return values from the web services api xml
+function _stanford_soe_r25_feed_get_value(&$results, $name, $key) {
```

Comment:
doesnt this do the same as https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_array_get_nested_value/7.x